packages/framework-integration-tests/src/read-models/cart-read-model.ts

```diff
@@ -18,6 +19,16 @@ export class CartReadModel {
     public cartItemsIds?: Array<string>
   ) {}
 
+  public static beforeFn(filter: FilterFor<CartReadModel>, currentUser?: UserEnvelope): FilterFor<CartReadModel> {
+    if (!filter.id || filter.id?.eq !== 'before-fn-test') return filter
+    return { id: filter.id } as FilterFor<CartReadModel>
```

Comment:
🤔  Would this return be equivalent to `return filter`?

Reply:
You're right! thanks!!